.gitignore

```diff
@@ -60,3 +60,4 @@ Pipfile.lock
 # Common virtualenvs
 .venv/
 venv/
+tags
```

Comment:
Is this intentional?

Reply:
I use Ctags  with Vin to jump to function definitions, it makes a **tags** file, so I dropped the exclusion in my **.gitignore**.

I can remove it though if may cause problems, or if it shouldnt be committed.

Reply:
Yeah, I'm gonna go ahead and remove **tags**
Just in case some day a valid **tags** file is created, i dont want anyone to have to suffer by accident :)